org.eclipse.jdt.junit/src/org/eclipse/jdt/internal/junit/ui/JUnitMessages.properties

```diff
@@ -190,6 +190,8 @@ JUnitContainerWizardPage_wizard_title=JUnit Library
 JUnitContainerWizardPage_option_junit3=JUnit 3
 JUnitContainerWizardPage_option_junit4=JUnit 4
 JUnitContainerWizardPage_option_junit5=JUnit 5
+JUnitContainerWizardPage_enableVintage=Enable <a>Vintage Engine</a>
+JUnitContainerWizardPage_enableVintage_tooltip=JUnit Vintage provides a TestEngine for running JUnit 3 and JUnit 4 based tests on the platform.
```

Comment:
> on the platform

Can we change it to "on JUnit 5"?

With "platform" everyone understands something different...

Reply:
Changed it to "JUnit Platform", because [JUnit 5 = JUnit 5 = JUnit Platform + JUnit Jupiter (+ JUnit Vintage)](https://junit.org/junit5/docs/current/user-guide/#overview-what-is-junit-5), claiming it runns Junit 3/4 on Junit 5 is confusing it think as it is not the case...

Reply:
But that's the case, we are running JUnit 3/4 on some libraries provided by JUnit 5. Anyway, I personally don't care, just it will confuse anyone else.

Reply:
Well technically there is `JUnit Platform` that defines generic test runs, then there is

- `JUnit Jupiter` as an Provider of a `JUnit Platform` (often refereed to as JUnit5) with new annotations and extension model
- `JUnit Vintage` what Implements a `JUnit Platform` to run Junit4/3 programming model

So basically yes we probabbly better have had not "JUNIT5" but `JUnit Platform` and then one would have ben able to dynamically select ANY `JUnit Platform` provider dynamical but thats a bit of history now and not sure if anyone is willing to change this just for the naming...

That's also why I said I'm not sure if such a configuration page is the right place to tell people JUnit concepts... I would assume that a user made the choice before open the wizard at all...